crypto/bn/bn_prime.c

```diff
@@ -250,6 +250,12 @@ int ossl_bn_check_prime(const BIGNUM *w, int checks, BN_CTX *ctx,
     return bn_is_prime_int(w, checks, ctx, do_trial_division, cb);
 }
 
+int ossl_bn_check_prime_fast(const BIGNUM *w, int checks, BN_CTX *ctx,
```

Comment:
Why not just rename `bn_is_prime_int` to `ossl_bn_check_prime_fast` and avoid the wrapper?

Reply:
I will see if there are objections to the PR before changing this..

Reply:
What about calling the function ossl_bn_check_generated_prime() or something like that? To emphasize this should be used only with generated random primes. In that case I would keep the bn_is_prime_int() as is.

Reply:
@paulidale any objections?